packages/appeals-service-api/src/routes/v2/appeal-cases/spec.yaml

```diff
@@ -0,0 +1,94 @@
+paths:
```

Comment:
thoughts on using tags, little annoying needing to add to main spec and here, but it looks nicer to me

Reply:
I hadn't appreciated what they did in the UI. added in.

You don't have to add to the root, but if you do you can control order in the UI which is handy.